css/at-rules/media.json

```diff
@@ -1605,26 +1605,11 @@
               },
               "firefox": [
                 {
-                  "version_added": "49",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "layout.css.prefixes.device-pixel-ratio-webkit",
-                      "value_to_set": "true"
-                    }
-                  ],
-                  "notes": [
-                    "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
-                  ]
+                  "version_added": "63"
```

Comment:
If I understand it correctly, here's how this was implemented:

- Implemented by default in Firefox 63+ _(this is new)_.
- Implemented behind one flag (layout.css.prefixes.device-pixel-ratio-webkit) in Firefox 49+.
- Implemented behind two flags (layout.css.prefixes.device-pixel-ratio-webkit and layout.css.prefixes.webkit) in Firefox 45+.

I would represent these support ranges like this:

```diff
"firefox": [
+  {  
+    "version_added": "63",
+    "notes": [
+      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
+    ]
+  },
  {
    "version_added": "49",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.prefixes.device-pixel-ratio-webkit",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
    ]
  },
  {
    "version_added": "45",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.prefixes.webkit",
        "value_to_set": "true"
      },
      {
        "type": "preference",
        "name": "layout.css.prefixes.device-pixel-ratio-webkit",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "Implemented as an alias for for <code>-moz-device-pixel-ratio</code>."
    ]
  }
],
```